static/app/components/platformPicker.tsx

```diff
@@ -29,24 +29,24 @@ const PlatformList = styled('div')`
 
 type Category = typeof PLATFORM_CATEGORIES[number]['id'];
 
-type Props = {
+interface PlatformPickerProps {
   setPlatform: (key: PlatformKey | null) => void;
   defaultCategory?: Category;
   listClassName?: string;
-  listProps?: React.ComponentProps<typeof PlatformList>;
+  listProps?: React.HTMLAttributes<HTMLDivElement>;
```

Comment:
do we have a PlatformList prop we can use here? is the list a div element?

Reply:
Yes, it's a div element. Idk if it's worth it tbh, I would just not write it for now and when we do need it, we can add it